packages/desktop-client/src/components/common.tsx

```diff
@@ -162,19 +162,22 @@ type AnchorLinkProps = {
   style?: CSSProperties;
   activeStyle?: CSSProperties;
   children?: ReactNode;
+  noContextMenu?: boolean;
 };
 
 export function AnchorLink({
   to,
   style,
   activeStyle,
   children,
+  noContextMenu,
 }: AnchorLinkProps) {
   let match = useMatch({ path: to });
 
   return (
     <NavLink
       to={to}
+      onContextMenu={e => noContextMenu && e.preventDefault()}
```

Comment:
Did this end up doing anything? If not it would be nice to remove it & the `react-device-detect` dependency.